uint8 setpoint_type
float64 x
float64 y
float64 z
float64 yaw
